RileyLinkKit/ViewControllers/RileyLinkDeviceTableViewController.swift

```diff
@@ -145,7 +145,7 @@ public class RileyLinkDeviceTableViewController: UITableViewController {
                 }
             case .Awake:
                 switch device.pumpState?.awakeUntil {
-                case let until? where until < NSDate():
+                case let until? where until.timeIntervalSinceNow < 0:
```

Comment:
This was just a silly original implementation by me. No sense instantiating a new `NSDate` object for this.